initrd/bin/gpg-gui.sh

```diff
@@ -0,0 +1,244 @@
+#!/bin/sh
+#
+set -e -o pipefail
```

Comment:
The effect of this is that once one of the command in a pipe fails, the whole line fails.